packages/@aws-cdk/aws-lambda/test/integ.docker.ts

```diff
@@ -0,0 +1,48 @@
+import { App, Construct, Stack, StackProps } from '@aws-cdk/core';
+import * as crypto from 'crypto';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as lambda from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const assetPath = path.join(__dirname, 'python-lambda-handler');
+    new lambda.Function(this, 'Function', {
+      code: lambda.Code.fromDockerImage({
+        image: 'python:3.6',
+        assetPath, // this is /asset in the container
+        command: [
+          'pip', 'install',
+          '-r', '/asset/requirements.txt',
+          '-t', '/asset',
+        ],
+        sourceHash: calcSourceHash(assetPath),
```

Comment:
Yes, but I needed this custom calculation here because when the python dependencies are taken into account I cannot get a stable hash (my machine vs CI).

Reply:
Got it. Still you can just use `core.FileSystem.fingerprint()` for this.